src/visual.ts

```diff
@@ -525,114 +563,128 @@ export class DualKpi implements IVisual {
                 break;
         }
 
-        this.titleSize = DualKpi.DefaultTitleSizes[this.sizeCssClass];
-
-        this.updateViewport({
-            width: availableWidth,
-            height: availableHeight
-        });
+        return { chartSpaceBetween, chartTitleSpace, iconOffset };
+    }
 
-        let chartHeight = (availableHeight - (chartSpaceBetween + chartTitleSpace)) / 2;
-        let topChartAxisConfig = { min: data.settings.dualKpiAxis.topChartAxisMin, max: data.settings.dualKpiAxis.topChartAxisMax };
-        let bottomChartAxisConfig = { min: data.settings.dualKpiAxis.bottomChartAxisMin, max: data.settings.dualKpiAxis.bottomChartAxisMax };
+    private dropBottomChart({
```

Comment:
do you really want to call it "drop"? or "draw"?

Reply:
Hahah, I didn't see that, thank you very much!